src/explore-education-statistics-admin/src/services/apiDataSetVersionService.ts

```diff
@@ -55,6 +68,15 @@ const apiDataSetVersionService = {
       `/public-data/data-set-versions/${versionId}/mapping/locations`,
     );
   },
+  updateLocationsMapping(
+    versionId: string,
+    data: LocationsMappingUpdateRequest,
+  ): Promise<LocationsMappingUpdateResponse> {
+    return client.patch(
```

Comment:
just querying about the use of patch - assume this is to adhere to strict HTTP semantics since you're only updating partial data?

Reply:
yes, it's just a partial update so patch is the correct method.